RawVideo/VideoStreamRenderer.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Azure.Communication.Calling.WindowsClient;
+using System;
+using System.Threading.Tasks;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace RawVideo
+{
+    internal class VideoStreamRenderer
+    {
+        private readonly MediaElement mediaElement;
+        private readonly CallVideoStream stream;
+
+        public VideoStreamRenderer(CallVideoStream stream)
+        {
+            this.stream = stream;
+
+            mediaElement = new MediaElement
+            {
+                // Fit
+                Stretch = Stretch.Uniform,
+                AutoPlay = true
+            };
+
+            RelativePanel.SetAlignVerticalCenterWithPanel(mediaElement, true);
+            RelativePanel.SetAlignHorizontalCenterWithPanel(mediaElement, true);
+        }
+
+        public async Task StartPreviewAsync()
+        {
+            Uri uri = null;
```

Comment:
nit: no need to initialize to null, and other places  have similar style

Reply:
i will remove it